src/OrleansRuntime/Silo/Silo.cs

```diff
@@ -25,14 +23,15 @@
 using Orleans.Runtime.Scheduler;
 using Orleans.Runtime.Startup;
 using Orleans.Runtime.Storage;
+using Orleans.Runtime.TestHooks;
 using Orleans.Serialization;
 using Orleans.Storage;
 using Orleans.Streams;
 using Orleans.Timers;
-using Orleans.Runtime.TestHooks;
 
 namespace Orleans.Runtime
 {
+    using Microsoft.Extensions.DependencyInjection.Extensions;
```

Comment:
I think the convention of inner is a carry-over from `#include` in C++, but it's the StyleCop way. I can undo